launch/launch/actions/execute_process.py

```diff
@@ -230,6 +231,10 @@ def __init__(
         self.__prefix = normalize_to_list_of_substitutions(
             LaunchConfiguration('launch-prefix', default='') if prefix is None else prefix
         )
+        self.__override_filter = False if prefix is None else True
+        self.__prefix_filter = normalize_to_list_of_substitutions(
+            LaunchConfiguration('launch-prefix-filter', default='')
+        )
```

Comment:
@camm73 nit: since `self.__prefix_filter` is irrelevant when `prefix is not None`, then:
```suggestion
        self.__prefix_filter = normalize_to_list_of_substitutions(
            LaunchConfiguration('launch-prefix-filter', default='')
        ) if prefix is None else None
```
conveys the same amount of information.